lib/auth/scopes/joining/service.go

```diff
@@ -130,5 +198,5 @@ func (s *Server) UpdateScopedToken(ctx context.Context, req *scopedjoiningv1.Upd
 		return nil, trace.AccessDenied("user %q does not have permission to update scoped tokens", authzContext.User.GetName())
 	}
 
-	return (scopedjoiningv1.UnimplementedScopedJoiningServiceServer{}).UpdateScopedToken(ctx, req)
+	return nil, trace.NotImplemented("scoped tokens can not be updated")
```

Comment:
Why is this? Any plans on implementing this endpoint? Is there a point in performing RBAC anyway?

Reply:
I think we may want to implement this at some point to update usage limits, but in general I don't think a token should be modifiable after creation. Good call on RBAC being pointless here though :+1: